guardrail-pii-masking-guardrails-ai/Package.md

```diff
@@ -0,0 +1,16 @@
+# PII Masking Guardrails AI
+
+## Overview
+
+This policy provides the capability to mask personally identifiable information (PII) in AI-generated content using Guardrails AI. It allows for the detection and masking of specified PII entities in both request and response payloads. This integration helps ensure that AI applications comply with data protection regulations while preventing the exposure of sensitive information.
+
+## Usage
+
+This will be available to select when attaching mediation policies to an egress AI Proxy Component. The following policy parameters are available:
```

Comment:
```suggestion
This will be available to select when attaching mediation policies to an egress AI proxy component. The following policy parameters are available:
```